f5router/f5router.go

```diff
@@ -236,18 +239,17 @@ func (r *F5Router) VerifyPlanExists(planID string) error {
 	}
 
 	r.plansMap.lock.Lock()
+	defer r.plansMap.lock.Unlock()
 	if _, ok := r.plansMap.plans[planID]; !ok {
-		r.plansMap.lock.Unlock()
 		return errors.New("plan for plan_id: " + planID + " not found")
 	}
-	r.plansMap.lock.Unlock()
 
 	return nil
 }
 
 // Run start the F5Router controller
 func (r *F5Router) Run(signals <-chan os.Signal, ready chan<- struct{}) error {
-	r.logger.Info("f5router-starting")
+	r.logger.Info("starting")
```

Comment:
Let's leave all these as is and open an issue if we need to make the logs consistent

Reply:
Ya backed these out as well and I'll open a ticket.